### PR TITLE
Add npm-run-all

### PR DESCRIPTION
**Pull request by XhmikosR**
Allows for parallel execution and slightly cleaner scripts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
   "scripts": {
     "build": "node bin/build.js",
-    "lint": "npm run lint:src && npm run lint:lockfile && npm run lint:markdown",
+    "lint": "npm-run-all --parallel lint:*",
     "lint:lockfile": "lockfile-lint --allowed-hosts npm --allowed-schemes https: --empty-hostname false --type npm --path package-lock.json",
     "lint:markdown": "markdownlint \"**/*.md\" --ignore node_modules --config .markdownlint.js",
     "lint:src": "eslint --cache --cache-location node_modules/.cache/.eslintcache --report-unused-disable-directives .",
     "nyc": "nyc",
     "mocha": "mocha test test/package-managers/npm test/package-managers/yarn && mocha --exit test/timeout",
-    "test": "npm run lint:all && npm run mocha",
```
Comment (XhmikosR) [inline package.json:42]: This was broken BTW since there was no `lint:all` script

Comment (raineorshine) [reply package.json:42]: Oops :)

**Comment by raineorshine** [comment]
Cool. Does `npm-run-all --parallel` fail if any one of the commands failed? It wasn't clearly documented in the README.

**Comment by XhmikosR** [comment]
> 
> 
> Cool. Does `npm-run-all --parallel` fail if any one of the commands failed? It wasn't clearly documented in the README.

Yeah. If you want to fail after if all commands have failed you can use `--continue-on-error`.

**Comment by raineorshine** [comment]
> > Cool. Does `npm-run-all --parallel` fail if any one of the commands failed? It wasn't clearly documented in the README.
> 
> Yeah. If you want to fail after if all commands have failed you can use `--continue-on-error`.

Great.

If you they are run in parallel, won't it interleave the output?

**Comment by XhmikosR** [comment]
> Another question: If you they are run in parallel, won't it interleave the output?

If I understand you correctly, I guess it does. You can check the Actions run for this branch.

**Comment by raineorshine** [comment]
Looks like we have the `--aggregate-output` option to fix that.

https://github.com/mysticatea/npm-run-all/issues/36

**Comment by XhmikosR** [comment]
Oh, I wasn't aware of that at all. I'll update the branch later :)

**Pull request merged**